src/controllers/user.js

```diff
@@ -39,14 +39,19 @@ export const getById = async (req, res) => {
 
 export const getAll = async (req, res) => {
   // eslint-disable-next-line camelcase
-  const { first_name: firstName } = req.query
+  const { first_name: firstName, cohort: inCohort } = req.query
+
+  const whereData = {}
+  if (inCohort === 'false') {
+    whereData.cohort = null
```

Comment:
Have you tried running the code without this? 

The reason why I'm asking is that, we're defining `whereData` as an empty object, so `whereData.cohort` will be undefined. So it shouldn't limit any users to a specific cohort by default.

Unless the idea here is to search for users who are not in a cohort?

Reply:
yes the idea is to search for users not in a cohort so we can populate the availible students list we're soon to implement on the client side for https://github.com/boolean-uk/team-dev-client/issues/200

Reply:
Okay great! I'll approve this PR then